dmriprep/config/reports-spec.yml

```diff
@@ -91,6 +91,13 @@ sections:
       the final images are resampled using cubic B-Spline interpolation.
     static: false
     subtitle: Alignment of diffusion and anatomical MRI data (surface driven)
+  - bids: {datatype: figures, desc: eddy, suffix: dwi}
+    caption: Head-motion and eddy current corrected diffusion data
+    description: Geometrical distortions derived from the so-called Eddy-currents, 
```

Comment:
```suggestion
    description: Eddy current-induced geometric distortions  
```

Reply:
@slimnsour would to accept this suggestion?

Reply:
Yup, fixed in caba8c33655177bc1731edb88ab49a82ad17e213